options/options.c

```diff
@@ -972,7 +972,6 @@ static const m_option_t mp_opts[] = {
     {"no-ometadata", OPT_REMOVED("use --no-ocopy-metadata")},
     {"video-stereo-mode", OPT_REMOVED("removed, try --vf=stereo3d")},
     {"chapter", OPT_REMOVED("use '--start=#123' '--end=#124' (for chapter 123)")},
-    {"video-aspect", OPT_REPLACED("video-aspect-override")},
```

Comment:
You could change this to `OPT_REMOVED` instead.

Reply:
I think it can even stay `OPT_REPLACED`, as I understand it will be redirected to new one.

Reply:
done